test/tests/menubar_menubar-2.js

```diff
@@ -305,6 +319,26 @@ ariaTest('Test aria-disabled="false" for all submenu role="menuitem"s',
       await menuitem[0].sendKeys(Key.ARROW_DOWN);
       await menuitem[1].sendKeys(Key.ENTER);
 
+      await menus[3].sendKeys(Key.ARROW_DOWN);
+      await menuitem[0].sendKeys(Key.ARROW_DOWN);
+      await menuitem[1].sendKeys(Key.ENTER);
+
+      await menus[3].sendKeys(Key.ARROW_DOWN);
+      await menuitem[0].sendKeys(Key.ARROW_DOWN);
+      await menuitem[1].sendKeys(Key.ENTER);
+
+      await menus[3].sendKeys(Key.ARROW_DOWN);
+      await menuitem[0].sendKeys(Key.ARROW_DOWN);
+      await menuitem[1].sendKeys(Key.ENTER);
+
+      await menus[3].sendKeys(Key.ARROW_DOWN);
+      await menuitem[0].sendKeys(Key.ARROW_DOWN);
+      await menuitem[1].sendKeys(Key.ENTER);
+
+      await menus[3].sendKeys(Key.ARROW_DOWN);
+      await menuitem[0].sendKeys(Key.ARROW_DOWN);
+      await menuitem[1].sendKeys(Key.ENTER);
+
```

Comment:
After looking into this a little, what's happening is `session.wait` will poll the DOM until the first argument passed to it is truthy, and a side effect of that is if you pass a function first, the function will execute until either it returns true, or the timeout is reached. It seems a bit fragile (as evidenced by breaking in this PR), and I'm not sure the best solution is to add more logic to the function itself.

A better solution might be to directly select X-Small/X-Large once instead of needing to select Smaller/Larger multiple times. I've copied a passing test below that does not rely on `t.context.session.wait`:

```
ariaTest('Test aria-disabled="false" for all submenu role="menuitem"s',
  exampleFile, 'submenu-menuitem-aria-disabled', async (t) => {
    // "aria-disable" should be set to false by default
    await assertAttributeValues(t, ex.submenuMenuitemSelector, 'aria-disabled', 'false');

    const menus = await t.context.session.findElements(By.css(ex.menubarMenuitemSelector));
    const sizeMenu = await t.context.session.findElement(By.css('[aria-label="Size"]'));
    const menuitems = await sizeMenu.findElements(By.css('[role="menuitem"]'));
    const menuItemRadios = await sizeMenu.findElements(By.css('[role="menuitemradio"]'));

    // select X-Small size
    await menus[3].sendKeys(Key.ARROW_DOWN);
    await menuItemRadios[0].sendKeys(Key.ENTER);

    const disabledFirstItem = await menuitems[0].getAttribute('aria-disabled');

    // Test that the item was successfully disabled
    t.true(
      disabledFirstItem === 'true',
      'The first menuitem in the last dropdown should become disabled after X-Small is selected'
    );

    // Select the X-Large size
    await menus[3].sendKeys(Key.ARROW_DOWN);
    await menuItemRadios[menuItemRadios.length - 1].sendKeys(Key.ENTER);

    const disabledSecondItem = await menuitems[1].getAttribute('aria-disabled');

    // Test that the item was successfully disabled
    t.true(
      disabledSecondItem === 'true',
      'The second menuitem in the last dropdown should become disabled after X-Large is selected'
    );
  });
```

Reply:
@smhigley this code is a more straightforward test, so I updated the pull request to use your suggested code